pom.xml

```diff
@@ -19,6 +19,11 @@
         <!--examples-->
         <module>examples/processors</module>
         <module>docker</module>
+        <!-- test jar builders -->
+        <module>streams/service/src/test/resources/custom-udf-microtest</module>
```

Comment:
@HeartSaVioR , thanks for the review.

Regarding the custom-udf-microtest module:  I agree the location of the jar-creation module seems hackey.  I was trying to achieve:
- Keep the source code for the test jar closely associated with the test jar itself (unlike the pre-built objects typically used for such purposes).
- But have the built jar end up under streams/service/src/test/resources/ without needing a copy phase.

The approach I took demonstrably works, and leaves the jar-creating source code closely tied to the jar itself, which I think is a big plus.  But it looks weird.

I'll re-organize it to a top-level module and see if you like it better.  We'll then be able to move it wherever you think it fits best.